app/services/doc_auth/mock/yml_loader_concern.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module DocAuth
+  module Mock
+    module YmlLoaderConcern
+      extend ActiveSupport::Concern
+      # @param [String] uploaded_file: string content
+      def parse_yaml(uploaded_file)
```

Comment:
Where was parse_yaml coming from before this PR?

Reply:
@charleyf it's not there, it was in the mock result_response, which is what happens at the third call of the conversation, too late to trigger errors in the first two calls of the three call conversation.